examples/components/responsive-navbar-view/responsive-navbar-view.component.jsx

```diff
@@ -15,11 +14,14 @@ const ResponsiveNavbarView = (props) => {
   const activeKey = 2;
 
   return (
-    <ResponsiveNavbar
-      activeKey={activeKey}
-      list={list}
-      onSelect={(href) => { props.router.push(href); }}
-    />
+    <div style={{ marginTop: '10px' }}>
```

Comment:
Is it possible to define styles in a separate scss file?

Reply:
yes, but was thinking this would not matter for example code

Reply:
ok, it's not a big deal